plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcIntegrationSmokeTest.java

```diff
@@ -120,6 +121,40 @@ public void testUnknownTypeAsVarchar()
         }
     }
 
+    @Test
+    public void testTableWithOnlyUnsupportedColumns()
+    {
+        Session session = Session.builder(getSession())
+                .setSchema("public")
+                .build();
+        try (TestTable table = new TestTable(getSqlExecutor(), "unsupported_table", "(geometry_column GEOMETRY)", ImmutableList.of("NULL", "'POINT(7 52)'"))) {
+            // SELECT all tables to avoid any optimizations that could skip the table listing
+            assertThat(getQueryRunner().execute("SELECT table_name FROM information_schema.tables").getOnlyColumn())
+                    .contains(table.getName());
+            assertQuery(
+                    format("SELECT count(*) FROM information_schema.tables WHERE table_name = '%s'", table.getName()),
+                    "SELECT 1");
+            assertQuery(
+                    format("SELECT count(*) FROM information_schema.columns WHERE table_name = '%s'", table.getName()),
+                    "SELECT 0");
+            assertQuery(
+                    session,
+                    format("SHOW TABLES LIKE '%s'", table.getName()),
+                    format("SELECT '%s'", table.getName()));
+            String unsupportedTableErrorMessage = "Table 'public.*' has no supported columns.*";
+            assertQueryFails(
+                    session,
+                    "SELECT * FROM " + table.getName(),
+                    unsupportedTableErrorMessage);
+            assertQueryFails(
+                    session,
+                    "SHOW CREATE TABLE " + table.getName(),
+                    unsupportedTableErrorMessage);
+            assertQueryReturnsEmptyResult(session, "SHOW COLUMNS FROM " + table.getName());
+            assertQueryReturnsEmptyResult(session, "DESC " + table.getName());
```

Comment:
Please use `DESCRIBE` (here and in commit message) as `DESC` is an undocumented alias.